packages/overlay/src/vaadin-overlay-stack-mixin.js

```diff
@@ -4,12 +4,14 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 
+const attachedInstances = [];
```

Comment:
Could be a set, unless we need the order of elements at some point.? That would make _appendAttachedInstance a bit safer.

Reply:
Thanks, changed.